community_components.md

```diff
@@ -47,3 +47,7 @@ Prospector permits automated collection of a wide range of metrics of open sourc
 useful in evaluating the project.
 Prospector is a sister project in CHAOSS, using Perceval (a GrimoireLab component) 
 for retrieving data from its data sources.
+
+## Scripts and miscellanenous utilities
+
+* [json2hat](https://github.com/lukaszgryglicki/bitergia_deployment/blob/master/cmd/json2hat/json2hat.go) by [Łukasz Gryglicki](https://github.com/lukaszgryglicki). Go script that imports the file github_users.json from the [cncf/gitdm](https://github.com/cncf/gitdm) GitHub repo into a SortingHat database.
```

Comment:
Now, `json2hat` lives in its own path: 
https://github.com/cncf/json2hat, see [this comment](
https://github.com/lukaszgryglicki/bitergia_deployment/issues/1#issuecomment-455759244)